mmcv/runner/hooks/logger/tensorboard.py

```diff
@@ -24,7 +23,7 @@ def __init__(self,
     @master_only
     def before_run(self, runner):
         super(TensorboardLoggerHook, self).before_run(runner)
-        if (LooseVersion(TORCH_VERSION) < LooseVersion('1.1')
+        if (digit_version(TORCH_VERSION) < digit_version('1.1')
```

Comment:
should test parrots related comparisons, `digit_version("parrots") < digit_version('1.1')` etc.